error-prone-contrib/src/main/java/tech/picnic/errorprone/bugpatterns/DirectReturn.java

```diff
@@ -67,7 +69,8 @@ public DirectReturn() {}
   @Override
   public Description matchBlock(BlockTree tree, VisitorState state) {
     List<? extends StatementTree> statements = tree.getStatements();
-    if (statements.size() < 2) {
+    ClassTree enclosingNode = ASTHelpers.findEnclosingNode(state.getPath(), ClassTree.class);
```

Comment:
One thing I considered is implementing `matchClass` and do this check there, but I think this makes more sense. CC @Stephan202.